### PR TITLE
Refs #13431 - Apache changes for pulp 2.8

### PR DESCRIPTION
**Pull request by parthaa**
Includes updates for pulp.conf

### CODE REVIEW

File: templates/httpd_pub.erb [hunk 1]
```diff
@@ -3,5 +3,10 @@ alias /pub /var/www/html/pub
 <Location /pub>
   PassengerEnabled off
   Options +FollowSymLinks +Indexes
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
```
Comment (ehelms) [inline templates/httpd_pub.erb]: You will need to set this up -- for example: https://github.com/Katello/puppet-katello/blob/c12447f1c01ba837d5ddbe1aeb908e6d18e6ab57/manifests/config.pp#L4

File: templates/pulp.conf.erb [hunk 2]
```diff
@@ -38,9 +46,25 @@ WSGIImportScript /srv/pulp/webservices.wsgi process-group=pulp application-group
...
+    SSLRequireSSL
+    Options +Indexes
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
+     Require all granted
+<% else -%>
+     Allow from all
+<% end -%>
+</Location>
+
 <%- if scope.lookupvar('operatingsystem') == 'Fedora' && scope.lookupvar('operatingsystemrelease') == '18' -%>
```
Comment (ehelms) [inline templates/pulp.conf.erb]: You could just remove lines 61-73

**Comment by parthaa** [comment]
@ehelms updated

**Comment by ehelms** [comment]
Looks good -- just need to get tests passing

File: manifests/config.pp [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # Capsule Config
 class capsule::config {
+  $apache_version = $::apache::apache_version
```
Comment (jlsherrill) [inline manifests/config.pp]: i don't think it needs to be in config.pp  just init.pp

**Comment by ehelms** [comment]
ACK - thanks @parthaa

**Comment by jlsherrill** [comment]
Merging thanks @parthaa

**Pull request merged**